### ISSUE TITLE
Problems with Traccar and openGTS after changing database user

### ISSUE THREAD

**Issue by Chris10014** [tananaev/traccar#1432]
Hello,
I had a running system with Traccar together with openGTS database.

Now I tried to change user and passwort of the database in traccar.cfg. That did not work. So I changed back to my old traccar.cfg and now I don't get positiondata from my tracker anymore into the database. I restarted traccar but it didn't help. For all other operation with the database, the user and passwort still work.

Could you give me any advice where and how I could find the failure?

**Comment by Chris10014** [comment]
Hello again,

I think I got the problem. There is a table missed in my database.

**Comment by tananaev** [comment]
How did it work before if a table was missing?

**Comment by Chris10014** [comment]
I think I deleted the table during I worked on my database. It was the Device table.
Best gegarte,
Christoph

----- Reply message -----
Von: "Anton Tananaev" <notifications@github.com>
An: "tananaev/traccar" <traccar@noreply.github.com>
Cc: "Chris10014" <clansche@t-online.de>
Betreff: [traccar] Problems with Traccar and openGTS after changing database user (#1432)
Datum: So., Sep. 27, 2015 22:23

How did it work before if a table was missing?

&mdash;
